sim/vcs.wake

```diff
@@ -53,12 +57,12 @@ def compileInputs options =
 
   def includeSources = map (sources _ `.*`) includeDirs | flatten
 
-  mkdir tmpDir, mkdir outputDir, includeSources ++ srcFiles
+  source vcsWrapperScript, mkdir tmpDir, mkdir outputDir, includeSources ++ srcFiles
 
 global def doVCSCompile options =
   def outputDir = options.getVCSCompilePlanOutputDir
-  def cmdline = "vcs", options.compileCmdlineArgs
-  def inputs = options.compileInputs
+  def cmdline = options.compileCmdlineArgs
+  def inputs = source vcsWrapperScript, options.compileInputs
```

Comment:
Am I reading this wrong, or are you adding the `vcs_wrapper` script twice? I see it both here and on line 60, in the `def compileInputs` function.

Reply:
you're right this is redundant. fixed